pkg/lightning/config/config.go

```diff
@@ -240,19 +232,25 @@ type CSVConfig struct {
 }
 
 type MydumperRuntime struct {
-	ReadBlockSize    ByteSize         `toml:"read-block-size" json:"read-block-size"`
-	BatchSize        ByteSize         `toml:"batch-size" json:"batch-size"`
-	BatchImportRatio float64          `toml:"batch-import-ratio" json:"batch-import-ratio"`
-	SourceDir        string           `toml:"data-source-dir" json:"data-source-dir"`
-	CharacterSet     string           `toml:"character-set" json:"character-set"`
-	CSV              CSVConfig        `toml:"csv" json:"csv"`
-	MaxRegionSize    ByteSize         `toml:"max-region-size" json:"max-region-size"`
-	Filter           []string         `toml:"filter" json:"filter"`
-	FileRouters      []*FileRouteRule `toml:"files" json:"files"`
-	NoSchema         bool             `toml:"no-schema" json:"no-schema"`
-	CaseSensitive    bool             `toml:"case-sensitive" json:"case-sensitive"`
-	StrictFormat     bool             `toml:"strict-format" json:"strict-format"`
-	DefaultFileRules bool             `toml:"default-file-rules" json:"default-file-rules"`
+	ReadBlockSize    ByteSize            `toml:"read-block-size" json:"read-block-size"`
+	BatchSize        ByteSize            `toml:"batch-size" json:"batch-size"`
+	BatchImportRatio float64             `toml:"batch-import-ratio" json:"batch-import-ratio"`
+	SourceDir        string              `toml:"data-source-dir" json:"data-source-dir"`
+	CharacterSet     string              `toml:"character-set" json:"character-set"`
+	CSV              CSVConfig           `toml:"csv" json:"csv"`
+	MaxRegionSize    ByteSize            `toml:"max-region-size" json:"max-region-size"`
+	Filter           []string            `toml:"filter" json:"filter"`
+	FileRouters      []*FileRouteRule    `toml:"files" json:"files"`
+	CaseSensitive    bool                `toml:"case-sensitive" json:"case-sensitive"`
+	StrictFormat     bool                `toml:"strict-format" json:"strict-format"`
+	DefaultFileRules bool                `toml:"default-file-rules" json:"default-file-rules"`
+	IgnoreColumns    map[string][]string `toml:"ignore-columns" json:"ignore-columns"`
```

Comment:
should use `[]IgnoreColumns` here?

Reply:
no, we need specify `db.table` for this config.